sync_radar_data_globus.py

```diff
@@ -148,7 +143,10 @@ def sanity_check(self):
             sys.exit("Data type can only be one of {}. Exiting".format(self.possible_data_types))
         # Note we cannot check the path here with typical os.is_path
         # since we may be running this script on a different machine than the destination endpoint,
```

Comment:
Do you document an example where you can have a different endpoint on another machine? I am just curious, I would like to know how that is possible for future work :)

Reply:
Haven't documented an example of this, but PERSONAL_UUID would need to have an option to be passed as argument, and to take precedence over the local one